lib/components/form/submissions.vue

```diff
@@ -131,14 +178,51 @@ export default {
           this.$nextTick(() => this.$refs.downloadLink.click());
         })
         .catch(() => {});
-    },
-    submitter(submission) {
-      const { submitter } = submission;
-      return submitter != null ? submitter.displayName : '';
-    },
-    createdAt(submission) {
-      return moment.utc(submission.createdAt).format('MMM D, Y H:mm:ss UTC');
     }
   }
 };
 </script>
+
+<style lang="sass">
+@import '../../../assets/scss/variables';
+
+#form-submissions-table1 {
+  float: left;
+  width: unset;
+
+  > thead > tr > th:last-child {
+    border-right: $border-bottom-table-heading;
+  }
+
+  > tbody > tr > td:last-child {
+    border-right: $border-top-table-data;
+  }
+}
+
+#form-submissions-table2-container {
+  overflow-x: scroll;
```

Comment:
Take the bottom margin off the table and put it on the container so that the margin is below the scrollbar, not above it.